Tests/Data/HydroMechanics/StaggeredScheme/InjectionProduction1D/RerefenceSolutionByMonolithicScheme/InjectionProduction1DMono.prj

```diff
@@ -36,6 +35,8 @@
                 <secondary_variable type="static" internal_name="velocity" output_name="velocity"/>
             </secondary_variables>
             <specific_body_force>0 0</specific_body_force>
+            <fluid_type>compressible_fluid</fluid_type>
+            <fluid_compressibility>3.33333333333e-5</fluid_compressibility>
```

Comment:
Just curious, what is this number? Specific fluid's value?

Reply:
this is just the calculated fluid compressibility to get the same specific storage as before. In a later Version when there is also the possibility to directly specify the specific storage this can be changed back